packages/sites/src/update-site.ts

```diff
@@ -21,11 +21,13 @@ import { updateItem, IUpdateItemResponse } from "@esri/arcgis-rest-portal";
  * @param {Object} model Site Model to update
  * @param {Array} patchList Array of property paths to update
  * @param {IHubRequestOptions} hubRequestOptions
+ * @param {boolean} [updateVersions=true] Optionally update the versions, defaults to true
  */
 export function updateSite(
   model: IModel,
   patchList: string[],
-  hubRequestOptions: IHubRequestOptions
+  hubRequestOptions: IHubRequestOptions,
+  updateVersions: boolean = true
```

Comment:
is there another way we can achieve the same w/o adding a trailing boolean argument? We try to have request options as the final argument. Maybe have `IUpdateSiteOptions extends IHubRequestOptions` that has `updateVersions` in it (also should ultimately have `patchList` in it too)?

Reply:
+1 for `IUpdateSiteOptions` vs trailing boolean

Reply:
I was going for a non-breaking change to avoid a major bump, but agree, the proposed is better. Looks reasonably easy to account for in Hub & Solutions.js.